Assets/Scripts/Systems/Modules/ModuleSlot.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace Systems.Modules
+{
+    /// <summary>
+    ///     Represents different module types
+    /// </summary>
+    [Flags]
+    [Serializable]
+    public enum ModuleType
+    {
+        None,
+        Weapon,
+        Engine
```

Comment:
These need to be set to `0`, `1 << 0`, `1 << 1`, ect. for flags to work. Since all `[Flags]` does is let you use bit-wise operators on the enum, you need to make sure they are bitmasks.